README.md

```diff
@@ -69,6 +69,6 @@ outline init
 
 Detailed information regarding how to contribute to Outline development, can be found in [CONTRIBUTING.md](./CONTRIBUTING.md). **Expect this information to be refined soon.**
 
-Please see [./docs/README.md](./docs/README.md) for information about developing for Outline.
+Please see [./docs/README.md](.packages/outline-docs/src/guides/development/component-development) for information about developing for Outline.
```

Comment:
Since this isn't pointing to `./docs/README.md` any more, we should correct the named portion of the link to be more relevant. Like `[Component Development Guide]` instead.

Reply:
updated